packages/lib/server-only/document/find-documents.ts

```diff
@@ -62,6 +62,18 @@ export const findDocuments = async ({
           Recipient: {
             some: {
               email: user.email,
+              role: {
+                not: RecipientRole.CC,
+              },
+            },
+          },
+        },
+        {
+          status: ExtendedDocumentStatus.COMPLETED,
+          Recipient: {
+            some: {
+              email: user.email,
+              role: RecipientRole.CC,
```

Comment:
These two filters seem weird, unless I'm tripping

Both:
- Check for a completed document
- Check that one of the recipients is the current user

And then they check for where the current user is:
- Non CC
- CC

Which would mean that it could be any role?